modules/claims_api/app/controllers/claims_api/docs/metadata_controller.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+module ClaimsApi
+  module Docs
+    class MetadataController < ::ApplicationController
+      skip_before_action(:authenticate)
+
+      def index
+        render json: {
+          meta: [
+            {
+              version: 'V1',
```

Comment:
I think the version should follow semantic versioning like 1.2.1 and the path be v<major version number> but the version not actually include the letter v when we reference it

Reply:
I went ahead and made the change.